pennylane/ops/op_math/controlled_ops.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains controlled operators based on the Controlled and ControlledOp class.
+"""
+
+import warnings
+from typing import Iterable
+
+from pennylane.operation import AnyWires
+from pennylane.ops.qubit.matrix_ops import QubitUnitary
+from .controlled_class import ControlledOp
+
+
+class ControlledQubitUnitary(ControlledOp):
+    r"""ControlledQubitUnitary(U, control_wires, wires, control_values)
+    Apply an arbitrary fixed unitary to ``wires`` with control from the ``control_wires``.
+
+    In addition to default ``Operation`` instance attributes, the following are
+    available for ``ControlledQubitUnitary``:
+
+    * ``control_wires``: wires that act as control for the operation
+    * ``control_values``: the state on which to apply the controlled operation (see below)
+    * ``target_wires``: the wires the unitary matrix will be applied to
+    * ``active_wires``: Wires modified by the operator. This is the control wires followed by the target wires.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (2,)
+    * Gradient recipe: None
+
+    Args:
+        U (Union[array[complex], QubitUnitary]): square unitary matrix or a QubitUnitary operation
+        control_wires (Union[Wires, Sequence[int], or int]): the control wire(s)
+        wires (Union[Wires, Sequence[int], or int]): the wire(s) the unitary acts on (optional if U is provided as a QubitUnitary)
+        control_values (List[int, bool]): a list providing the state of the control qubits to control on (default is the all 1s state)
+        unitary_check (bool): whether to check whether an array U is unitary when creating the operator (default False)
```

Comment:
`do_queue` missing

Reply:
Thanks - also in the Controlled docstring :) Updated now!